Answers.md

```diff
@@ -2,8 +2,14 @@
 
 1. What is React JS? How does it differ from other JavaScript frameworks?
 
+ReactJS is a frontend framework that allows you to create reusable pieces of code called components and use them in your application by mutating and changing state and passing props.  It differs from other frameworks because it allows you to create components and use state/props.
+
 
 2. Explain briefly the React Component Lifecycle. Name three of the methods that are a part of the lifecycle and what they do.
 
+ComponentDidMount() -
+Render() -
+
+

```

Comment:
Good job on question number one.  What do you think about the rest of the questions?